src/main/java/duke/parser/Parser.java

```diff
@@ -1,3 +1,7 @@
+//@@author pzaiming-reused
```

Comment:
It's great practice that you credited the code owner! But I think the instruction mentioned that you have to enclose the reused code block with the `//@@author` tags, meaning **both** the start and the end of the code block have to be marked!

```Java
//@@author {yourGithubUsername}-reused
//{Info about the source...}

{Reused code (possibly with minor modifications) here ...}

//@@author
```